README.md

```diff
@@ -617,130 +621,143 @@ Note that it is also possible to generate a synthetic dataset based on the entir
 
 There are many options for training the model with your synthetic data-enhanced dataset.
 
-> **Note:** **Every** `ilab` command needs to run from within your Python virtual environment.
+### ✋ Before you begin training
 
-#### ilab model training pipelines
+There are a few models you need to download before running the InstructLab end-to-end workflow locally.
 
-`ilab model train` has three pipelines: `simple`, `full`, and `accelerated`. The default is `accelerated`.
+- Download the `granite-7b-lab` model for training:
 
-1. `simple` uses an SFT Trainer on Linux and MLX on MacOS. This type of training takes roughly an hour and produces the lowest fidelity model but should indicate if your data is being picked up by the training process.
-2. `full` uses a custom training loop and data processing funcions for the granite family of models. This loop is optimizied for CPU and MPS functionality. Please use `--pipeline=full` in combination with `--device=cpu` (Linux) or `--device=mps` (MacOS). You can also use `--device=cpu` on a MacOS machine. However, MPS is optimized for better performance on these systems.
-3. `accelerated` uses the instructlab-training library which supports GPU accelerated and distributed training. The `full` loop and data processing functions are either pulled directly from or based off of the work in this library.
+   ```shell
+   ilab model download --repository intructlab/granite-7b-lab --hf-token <your-huggingface-token> 
+   ```
 
-#### Train the model locally
+- Download the `prometheus-8x7b-v2.0` for multi-phase training and benchmark evaluation:
 
-```shell
-ilab model train
-```
+   ```shell
+   ilab model download --repository prometheus-eval/prometheus-8x7b-v2.0 --hf-token <your-huggingface-token> 
+   ```
 
-> **NOTE:** ⏳ This step can potentially take **several hours** to complete depending on your computing resources. Please stop `ilab model chat` and `ilab model serve` first to free resources.
+#### InstructLab model training pipelines
```

Comment:
I love this and would love to see one for SDG as well!

Reply:
Im working on one in the SDG repo, but I need to understand them more lol